common/unified-chart/templates/high-priority/analysis-template.yaml

```diff
@@ -0,0 +1,32 @@
+{{- $top := . -}}
+{{- $values := .Values -}}
+{{- if .Values.argoRollouts.progressiveDelivery.enabled }}
+---
+apiVersion: argoproj.io/v1alpha1
+kind: AnalysisTemplate
+metadata:
+  name: {{ include "hp.name" . }}
+spec:
+  args:
+    - name: service-name
+      value: {{ include "hp.name" . }}
+  metrics:
+    - name: succesful-black-box-test
+      successCondition: result == true
+      provider:
+        web:
+        {{- with .Values.argoRollouts.progressiveDelivery }}
+          method: POST # valid values are GET|POST|PUT, defaults to GET
+          url: {{ .blackBox.url | default "http://nexus.default.svc.cluster.local/blackbox" }}
+          timeoutSeconds: {{ .blackBox.timeout | default "1200" }} # seconds
+          headers:
+            {{- range  $header := .blackBox.headers }}
+            - key: {{ $header.key }} # if body is a json, it is recommended to set the Content-Type
+              value: {{ $header.value }}
+            {{- end }}
+          jsonBody: # If using jsonBody Content-Type header will be automatically set to json
+            service-name: {{ include "hp.name" $top }}
+            version: {{ include "appVersion" $top }}
+        {{- end }}
+          jsonPath: "{$.status}"
```

Comment:
$ usually points to the root scope
i suggest using $values..........status

Reply:
@mqxter It's not a value rendered by the helm, this is used by the Argo rollouts analysis template to evaluate success conditions.